build-artifacts.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
```

Comment:
Where is this used? Is it here by mistake?

Reply:
Oh never mind. I just saw for commit message

Reply:
Yeah its used to actually locally build the artifacts. I will use this later to refactor the process for publishing things with Github Workflows.